lightning/src/ln/onion_utils.rs

```diff
@@ -2706,7 +2755,7 @@ impl AttributionData {
 
 	/// Verifies the attribution data of a failure packet for the given position in the path. If the HMAC checks out, the
 	/// reported hold time is returned. If the HMAC does not match, None is returned.
-	fn verify(&self, message: &Vec<u8>, shared_secret: &[u8], position: usize) -> Option<u32> {
+	fn verify(&self, message: &[u8], shared_secret: &[u8], position: usize) -> Option<u32> {
```

Comment:
Pre-existing, but it seems way more intuitive to return a `Result` here?

Reply:
Changed